packages/contracts-bedrock/src/Safe/VetoerSet/AddOwnerModule.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+import { ISemver } from "src/universal/ISemver.sol";
+
+import { OwnerGuard } from "./OwnerGuard.sol";
+
+/// @title AddOwnerModule
+/// @notice This module allows any specifically designated address to add owners to the
+///         Safe Account. Specifically, the Optimism Foundation may add new owners.
+contract AddOwnerModule is ISemver {
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.0
+    string public constant version = "1.0.0";
+
+    /// @notice The Safe contract instance
+    Safe internal immutable safe;
+
+    /// @notice The OwnerGuard instance
+    OwnerGuard internal immutable ownerGuard;
+
+    /// @notice The OP Foundation multisig address
+    address internal immutable opFoundation;
+
+    /// @notice Thrown when trying to add an owner through this module but the caller is not
+    ///         the whitelisted OP Foundation address.
+    /// @param sender The sender address.
+    error SenderIsNotOpFoundation(address sender);
+
+    /// @notice The module constructor.
+    /// @param safe_ The Safe Account address
+    /// @param ownerGuard_ The owner guard contract address.
+    /// @param opFoundation_ The OP Foundation multisig address.
+    constructor(Safe safe_, OwnerGuard ownerGuard_, address opFoundation_) {
```

Comment:
nit: the underscore is generally a prefix when its an argument

Reply:
I am definitely open to do the change but Solidity Style Guide, naming convention reserves `_` prefix [for private and internal symbols](https://docs.soliditylang.org/en/latest/style-guide.html#underscore-prefix-for-non-external-functions-and-variables) while `_` suffix is used in case of [naming collision](https://docs.soliditylang.org/en/latest/style-guide.html#avoiding-naming-collisions) (in normal circumstances function argument names should be named [without](https://docs.soliditylang.org/en/latest/style-guide.html#function-argument-names) `_`)

Reply:
@tynes I know we've discussed this item before, but just pointing out that this repo also uses trailing in some places :) https://github.com/ethereum-optimism/optimism/blob/d72fb46daf3a6831cb01a78931f8c6e0d52ae243/packages/contracts-bedrock/src/L1/DelayedVetoable.sol#L79